### PR TITLE
Rephrase the search XMLRPC disable error

### PR DESCRIPTION
**Pull request by pradyunsg**
Based on the idea from https://github.com/pypa/pip/issues/5216#issuecomment-1235329876.

### CODE REVIEW

File: warehouse/legacy/api/xmlrpc/views.py [hunk 1]
```diff
@@ -235,14 +235,11 @@ def search(request, spec: Mapping[str, Union[str, List[str]]], operator: str = "
...
+        domain = request.registry.settings.get("warehouse.domain", request.domain)
         raise XMLRPCWrappedError(
             RuntimeError(
-                (
-                    "PyPI's XMLRPC API is currently disabled due to "
-                    "unmanageable load and will be deprecated in the near "
-                    "future. See https://status.python.org/ for more "
-                    "information."
-                )
+                "PyPI does not support 'pip search' / XMLRPC search. "
```
[the remark this answers is not in the document]
Comment (brainwane) [reply warehouse/legacy/api/xmlrpc/views.py:241]: I'd love for us to link to an FAQ or documentation page here about the deprecation.

[the remark this answers is not in the document]
Comment (pradyunsg) [reply warehouse/legacy/api/xmlrpc/views.py:241]: > I'd love for us to link to an FAQ or documentation page here about the deprecation.

While I'm in agreement, I'd argue that an FAQ / documentation page can be added in a follow up -- I'd prefer to not increase the scope of this PR. :)

**Comment by pradyunsg** [comment]
Anything I can do to help this move forward?

**Review by di — APPROVED** [review]

**Review by dstufft — APPROVED** [review]

**Comment by pradyunsg** [comment]
Thanks for taking this over the line @ewdurbin!

**Pull request merged**